tests/codegen/call_nsecs.cpp

```diff
@@ -0,0 +1,14 @@
+#include "common.h"
+
+namespace bpftrace {
+namespace test {
+namespace codegen {
+
+TEST(codegen, call_nsecs)
+{
+  test("k:f { @x = nsecs(); }", NAME);
```

Comment:
I don't think that it's necessary to have a test for the `nsecs()` call since we already have a codegen test for `nsecs` as a builtin. Maybe rather add codegen test for `nsecs(tai)`?

Reply:
Update to patch: [add codegen test for nsecs()](https://github.com/iovisor/bpftrace/pull/2619/commits/edea0147485c4b1764b642b8c400625bd5d39053). Thanks.